src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
Yeah, that's common, but it's totally valid to have many `footer` elements throughout a page.

`page-footer` sounds like a good classname to me. 👍